cloverage/src/cloverage/source.clj

```diff
@@ -1,10 +1,13 @@
 (ns cloverage.source
   (:require [clojure.tools.reader :as r]
             [clojure.tools.reader.reader-types :as rt]
-            [clojure.java.io :as io]))
+            [clojure.java.io :as io]
+            [clojure.string :as str])
+  (:import (java.io InputStreamReader File PushbackReader)
+           (clojure.lang RT)))
 
 (defn- get-loader ^ClassLoader []
-  (clojure.lang.RT/baseLoader))
+  (RT/baseLoader))
```

Comment:
Wait, what does this change do? Isn't clojure.lang already always available?

Reply:
I moved the import to the top so you don't have to fully quantify the path, see https://clojuredocs.org/clojure.core/import :)

Reply:
Sure, sure -- I understand `:import`, just checking that there isn't some weird side effect that you need this for since AFAICT cloverage is mostly just a mechanism for finding weird edge cases about Clojure's dependency management :)